bson/src/main/org/bson/codecs/pojo/ClassModelBuilder.java

```diff
@@ -55,6 +55,7 @@
     private String discriminator;
     private String discriminatorKey;
     private String idPropertyName;
+    private boolean isRecord;
```

Comment:
I added this to avoid a call to `PojoBuilderHelper::isRecord` in `ConventionAnnotationImpl`. It's not added to the `ClassModel` that is built.